### PR TITLE
fix(gatsby-source-wordpress): match full urls when replacing media item links to static

### PR DESCRIPTION
**Pull request by TylerBarnes**
Usually file url's in WP don't include the full url and are just paths. A customer recently discovered that this is not the case for certain ACF fields.
I added an additional regexp here to catch that case. I didn't modify the original because I don't want to cause problems for any replacing that's already working and I'm not a regexp wizard.

### CODE REVIEW

File: packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js [hunk 1]
```diff
@@ -738,17 +738,32 @@ const replaceFileLinks = async ({
     return nodeString
   }
 
-  const hrefMatches = execall(
-    /(\\"|\\'|\()([^'"()]*)(\/wp-content\/uploads\/[^'">()]+)(\\"|\\'|>|\))/gm,
-    nodeString
-  )
+  const hrefMatches = [
```
Comment (wardpeet) [inline packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js:741]: can you add an example comment on what you're trying to match? That helps people to understand the regexes better.

Comment (kathmbeck) [reply packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js:741]: We could also add a unit test?

Comment (wardpeet) [reply packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js:741]: That's even better

Comment (TylerBarnes) [reply packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js:741]: Oh good call, there's already one there so I'll expand it a bit

Comment (TylerBarnes) [reply packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js:741]: Turns out there wasn't one. I ended up adding an integration test instead as well as some comments explaining what each regex is matching for

**Review by kathmbeck — APPROVED** [review]

**Pull request merged**